client/src/components/drawer/index.tsx

```diff
@@ -144,12 +143,16 @@ export default function ChronicleDrawer({
           </ListItemButton>
         )}
 
-        <ListItem button key="analytics">
+        <ListItemButton
+          component="a"
```

Comment:
Do we use `a` or the `Link` component here instead?

Reply:
This will create an a component with the abstraction and style the  menu item normally has it gives us full access to the entire api of `a` without the need to impact the structure of the component. So this model is better than using link or a directly

In the dom it will just be an `a` element